### DISCUSSION TITLE
inverter returns l1_voltage instead of p1_voltage

### DISCUSSION THREAD

**Discussion by CreasolTech** [addiejanssen/domoticz-solaredge-modbustcp-plugin#29] [Q&A] [answered]
Hi, 
I'm using this plugin since several month.
It stopped working in the last days, maybe for an automatic update of the solaredge firmware (I have a single-phase inverter).
The plugins checks parameters p1_voltage, p1_current, ... I had to change, in plugin.py, p1 with l1 .
Maybe the same modifications should be made replacing p2 and p3 prefixes with l2 and l3, or (much better), let plugin working with both p* and l* 
Below the parameters from my inverter:
`Inverter: inverter values: {

    "c_manufacturer": "SolarEdge",
    "c_model": "SE6000H-RW000BNN4",
    "c_version": "0004.0009.0033",
    "c_deviceaddress": 1,
    "c_sunspec_did": 101,
    "current": 314,
    "l1_current": 314,
    "l2_current": 0,
    "l3_current": 0,
    "current_scale": -2,
    "l1_voltage": 2297,
    "l2_voltage": 0,
    "l3_voltage": 0,
    "l1n_voltage": 0,
    "l2n_voltage": 0,
    "l3n_voltage": 0,
    "voltage_scale": -1,
    "power_ac": 7111,
    "power_ac_scale": -1,
    "frequency": 50028,
    "frequency_scale": -3,
    "power_apparent": 7216,
    "power_apparent_scale": -1,
    "power_reactive": 12264,
    "power_reactive_scale": -2,
    "power_factor": 9854,
    "power_factor_scale": -2,
    "energy_total": 20633248,
    "energy_total_scale": 0,
    "current_dc": 19524,
    "current_dc_scale": -4,
    "voltage_dc": 3698,
    "voltage_dc_scale": -1,
    "power_dc": 7220,
    "power_dc_scale": -1,
    "temperature": 2814,
    "temperature_scale": -2,
    "status": 4,
    "vendor_status": 0,
    "rrcr_state": 0,
    "active_power_limit": 100,
    "cosphi": 0
}
`
Thanks for your great work. I'm happy to test your new code.
Paolo

**Comment by addiejanssen** [comment]
Hi,

This is the result of a change in the solaredge_modbus library.
See release notes here: https://github.com/nmakel/solaredge_modbus/releases/tag/0.7.0

This plugin is tested up to version 0.6.2 and does not recognize the new register names.
I will add it as an issue to the plugin and may fix it at some point in time.

**Comment by addiejanssen** [comment, marked as answer]
As of version (1.1.1) of the plugin, the plugin will use and report on the "L" instead of the "P" values.
